pandas/core/arrays/timedeltas.py

```diff
@@ -776,12 +776,14 @@ def __rdivmod__(self, other):
         res2 = other - res1 * self
         return res1, res2
 
-    # Note: TimedeltaIndex overrides this in call to cls._add_numeric_methods
     def __neg__(self):
         if self.freq is not None:
             return type(self)(-self._data, freq=-self.freq)
         return type(self)(-self._data)
 
+    def __pos__(self):
+        return type(self)(self._data, freq=self.freq)
```

Comment:
This is new? Or where was it done before?

Reply:
In `_add_numeric_methods_unary`.